pages/linux/chroot.md

```diff
@@ -0,0 +1,15 @@
+# chroot
+
+> Run command or interactive shell with special root directory.
+
+- Run ls command as new root directory
+
+`chroot chroot {{/path/to/new/root}} {{ls}}`
```

Comment:
1. Need `chroot` once, you repeated it twice.
2. Should be either `ls` or `{{command}}`. If you choose `{{command}}`, then update the description.